CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [1.6.0] - 2024-05-29
```

Comment:
I know I asked you to add this yesterday but now since I'm planning to merge my branches down as well can you remove it?  :-P